activerecord-virtual_attributes.gemspec

```
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
 
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activerecord", "~> 6.1.0"
+  spec.add_runtime_dependency "activerecord", ">=6.1.7.8"
```

I think you should limit it to 7.0 as a maximum for now

```suggestion
  spec.add_runtime_dependency "activerecord", ">=6.1.7.8", "<7.1"
```

---

ok

---

done